report-03/threat model.wiki

```diff
@@ -169,6 +197,10 @@ This document outlines the OBPP threat model used to develop [[criteria.md|crite
 |Provide a deterministic build system that allows users to verify that the binary version they have received was compiled from the public source code
 |V D 2 b
 |-
+|Require personally identifying information in order to use the wallet
```

Comment:
mistake here: You accidentally copied this attack and placed the new attack (issue #63 ) under the "countermeasure" column